package-lock.json

```diff
@@ -21989,9 +21989,8 @@
       }
     },
     "eslint-config-expensify": {
-      "version": "2.0.18",
-      "resolved": "https://registry.npmjs.org/eslint-config-expensify/-/eslint-config-expensify-2.0.18.tgz",
-      "integrity": "sha512-jEBAcXWm89NptiprvlBrRoDsK8zJbbkt8yVZOgzo/Vadp86gL0oc7blgyLfJHA2cjecs1kgfPYGByXb6V8v8cw==",
+      "version": "2.0.19",
+      "resolved": "git+https://github.com/Expensify/eslint-config-expensify.git#afc71e84361cc68af4a85b425b3a599c4ac1861d",
```

Comment:
I think this needs to be updated. If I try to do `npm install` with the committed `package-lock.json`, I get this error:

```sh

npm ERR! code 128
npm ERR! Command failed: git checkout afc71e84361cc68af4a85b425b3a599c4ac1861d
npm ERR! fatal: reference is not a tree: afc71e84361cc68af4a85b425b3a599c4ac1861d
npm ERR! 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/aldocanepa/.npm/_logs/2021-11-24T18_57_59_383Z-debug.log
```

Deleting the `package-lock.json` and running `npm install` changes to:

```json
"eslint-config-expensify": {
    "version": "2.0.19",
    "resolved": "git+https://github.com/Expensify/eslint-config-expensify.git#e1edb2cfe7fd91b9b131187cb19481ba9d29705b",
    ...
}
```

Reply:
Oh hmm weird..